### PR TITLE
chore: deliver token list by chain id

### LINKED ISSUE #11: Deliver token list for each chain id
For now, we are using a single token list file for all chain ids. It's getting huge and making requests longer time to consume. Let's divide it into multiple files which each file will serve a single chain id.

### PR DESCRIPTION
**Pull request by NikoManiac**
close #11

### CODE REVIEW

File: scripts/build.sh [hunk 1]
```diff
@@ -1,15 +1,29 @@
...
 
 VERSION=$(node -p -e "require('./package.json').version")
+CHAIN[0]=dist/v$VERSION
+CHAIN[1]=dist/v$VERSION/1
+CHAIN[3]=dist/v$VERSION/3
+CHAIN[4]=dist/v$VERSION/4
+CHAIN[56]=dist/v$VERSION/56
+CHAIN[97]=dist/v$VERSION/97
+CHAIN[137]=dist/v$VERSION/137
+CHAIN[80001]=dist/v$VERSION/80001
```
Comment (UncleBill) [inline scripts/build.sh:11]:
```suggestion
for i in {1,3,4,56,97,137,80001}; do
  CHAIN[$i]=dist/v$VERSION/$i
done
```

**Review by guanbinrui — DISMISSED** [review]

**Comment by guanbinrui** [comment]
Should not merge it before the next version.

File: scripts/fetch-debank-logo-uri.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+async function fetchDebankLogoURI(chainId, tokenIds) {
+  if (!CHAINID_MAPPING[chainId]) return []
+
+  const allRequest = []
+  for (let i = 0, j = tokenIds.length; i < j; i += DEBANK_REQUEST_LIMIT) {
+    const currentIds = tokenIds.slice(i, i + DEBANK_REQUEST_LIMIT);
+    allRequest.push(() => fetch(getRequestURL(CHAINID_MAPPING[chainId], currentIds.join(','))).then(res => res.json()))
+  }
+  const data = (await Promise.allSettled(allRequest.map(x => x()))).filter(x => x.status === 'fulfilled').map(x => x.value).flat()
```
Comment (UncleBill) [inline scripts/fetch-debank-logo-uri.js:22]:
```suggestion
  for (let i = 0, j = tokenIds.length; i < j; i += DEBANK_REQUEST_LIMIT) {
    const currentIds = tokenIds.slice(i, i + DEBANK_REQUEST_LIMIT);
    allRequest.push(fetch(getRequestURL(CHAINID_MAPPING[chainId], currentIds.join(','))).then(res => res.json()))
  }
  const data = (await Promise.allSettled(allRequest)).filter(x => x.status === 'fulfilled').map(x => x.value).flat()
```

**Review by UncleBill — DISMISSED** [review]

File: scripts/build.sh (hunk 1 above)
Comment (NikoManiac) [reply scripts/build.sh]: 👍

File: scripts/fetch-debank-logo-uri.js (hunk 2 above)
Comment (NikoManiac) [reply scripts/fetch-debank-logo-uri.js]: done

**Review by guanbinrui — APPROVED** [review]

**Pull request merged**